### DISCUSSION TITLE
Non-contact grid on navier-stokes

### DISCUSSION THREAD

**Discussion by AdelineHunter** [idaholab/moose#21706] [Q&A Meshing] [answered]
Hi, 

May I ask if MOOSE can complete the flow calculation on the non_contact grid? In fact, for heat conduction problem MOOSE can solve on the non-contact grid.

![image](https://user-images.githubusercontent.com/108536058/181407772-87feef24-7c8f-4d2a-9840-bb967a13a224.png)

Thanks.

**Comment by GiudGiud** [comment, marked as answer]
Hello

no we don’t. We re adding support for it right now, but we re doing a radiation interface not a convection interface. 

You really should make a conforming mesh, it seems to me there is no reason not to here?

Guillaume
